app/assets/stylesheets/application.css.scss

```diff
@@ -54,7 +54,7 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 }
 
-#new-transfer {
+#new_transfer {
```

Comment:
Avoid using id selectors<br>Selector `new_transfer` should be written in lowercase with hyphens